istio-pilot-agent-1.19.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: istio-pilot-agent-1.19
+  version: 1.19.0
+  epoch: 0
+  description: Nanny binary for Istio Envoy
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - istio-pilot-agent=1.19.999
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+  environment:
+    CGO_ENABLED: "0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/istio/istio
+      tag: ${{package.version}}
+      expected-commit: 3b3ca8ec1632961e355f398f7357ebed9b13aa43
+
+  - uses: go/build
+    with:
+      packages: ./pilot/cmd/pilot-agent
+      output: pilot-agent
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/var/lib/istio/envoy
+      cp ./tools/packaging/common/envoy_bootstrap.json \
+        ${{targets.destdir}}/var/lib/istio/envoy/envoy_bootstrap_tmpl.json
+      cp ./tools/packaging/common/gcp_envoy_bootstrap.json \
+        ${{targets.destdir}}/var/lib/istio/envoy/gcp_envoy_bootstrap_tmpl.json
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-compat
+    pipeline:
+      - runs: |
+          # link /usr/local/bin/pilot-agent -> /usr/bin/pilot-agent to match
+          # what the Istio Helm charts may expect.
+          mkdir -p ${{targets.subpkgdir}}/usr/local/bin
+          ln -sf /usr/bin/pilot-agent ${{targets.subpkgdir}}/usr/local/bin/pilot-agent
+    dependencies:
+      provides:
+        - istio-pilot-agent-compat=1.19.999
```

Comment:
You don't need `.999` if this didn't exist in the pre version-stream package.

I think @vaikas also has a new syntax for this?

Reply:
Thanks @vaikas  🙏  https://github.com/chainguard-dev/melange/blob/main/docs/BUILD-FILE.md#provides

Reply:
Done